Makefile

```diff
@@ -79,8 +79,8 @@ CGO_LDFLAGS_DYN = -L$(abspath $(LIBBPF_DIR)) -fuse-ld=ld -lelf -lz -lbpf
 
 # libbpf build flags:
 # CFLAGS = -g -O2 -Wall -fpie
-CFLAGS ?= -g -O2 -Werror -Wall -std=gnu89 # default CFLAGS
-LDFLAGS ?= -fuse-ld=lld
+CFLAGS ?= -g -O2 -Werror -Wall -std=gnu89 -fpic # default CFLAGS
```

Comment:
This was the fix. I wonder why I had removed it from the flags, even if it was in the default flags. 
fPIC and fPIE are practically the same things.